WordPress/Classes/ViewRelated/NUX/SignupEpilogueCell.swift

```diff
@@ -64,27 +72,52 @@ class SignupEpilogueCell: UITableViewCell {
     // MARK: - Public Methods
 
     func configureCell(forType newCellType: EpilogueCellType,
-                       labelText: String,
+                       labelText: String? = nil,
                        fieldValue: String? = nil,
                        fieldPlaceholder: String? = nil) {
+
         cellType = newCellType
+
         cellLabel.text = labelText
         cellLabel.textColor = .text
 
         cellField.text = fieldValue
-        cellField.textColor = .text
         cellField.placeholder = fieldPlaceholder
         cellField.delegate = self
-        cellField.isSecureTextEntry = (cellType == .password)
+
+        configureForPassword()
+
         selectionStyle = .none
 
         configureAccessoryType(for: newCellType)
         configureTextContentTypeIfNeeded(for: newCellType)
         configureAccessibility(for: newCellType)
+
+        addBottomBorder(withColor: .divider)
+
+        // TODO: remove this when `WordPressAuthenticatorStyle:textFieldBackgroundColor` is updated.
+        // This background color should be inherited from LoginTextField.
+        // However, since the Auth views haven't been updated, the color is incorrect.
+        // So for now we'll override it here.
+        cellField.backgroundColor = .basicBackground
     }
 
     // MARK: - Private behavior
 
+    private func configureForPassword() {
+        let isPassword = (cellType == .password)
+        cellLabel.isHidden = isPassword
+
+        cellField.isSecureTextEntry = isPassword
+        cellField.showSecureTextEntryToggle = isPassword
+        cellField.textAlignment = isPassword ? .left : .right
+        cellField.textColor = isPassword ? .text : .textSubtle
+
+        cellFieldLeadingConstraintWithLabel.isActive = !isPassword
+        cellFieldLeadingConstraintWithoutLabel.isActive = isPassword
```

Comment:
Just a question/suggestion: would it be easier to use a stack view to handle the case of a label hidden/not hidden instead of explicitly manipulate constraints?

Reply:
You'd think so wouldn't you? Well, I tried. 😄 But because of the behind the scenes magic done with the custom field in WPAuth, it did _not_ go well.

FWIW, I firmly believe after we update `LoginTextField` (the custom field) for the new auth design, this epilogue will have to be adjusted anyway. If we're clever about it, maybe it'll end up being smarter.

Reply:
Oh I see now! Thanks for clarifying!!